index.js

```diff
@@ -0,0 +1,221 @@
+                Entrypoint: '/bin/true',
+                Labels: {
+                    sdbuild: config.buildId
+                }
+            })
+            .then(launchContainer => Promise.all([
+                this._createContainer({
+                    name: `${config.buildId}-log`,
+                    Image: `screwdrivercd/log-service:${this.logVersion}`,
+                    Entrypoint: '/opt/screwdriver/tini',
+                    Labels: {
+                        sdbuild: config.buildId
+                    },
+                    Cmd: [
+                        '--',
+                        '/opt/screwdriver/logservice',
+                        '--emitter',
+                        '/opt/screwdriver/emitter',
+                        config.buildId
+                    ],
+                    Env: [
+                        `SD_TOKEN=${config.token}`
+                    ],
+                    HostConfig: {
+                        Memory: 200 * 1024 * 1024,
+                        MemoryLimit: 300 * 1024 * 1024,
+                        VolumesFrom: [
+                            `${launchContainer.id}:rw`
+                        ]
+                    }
+                }),
+                this._createContainer({
+                    name: `${config.buildId}-build`,
+                    Image: config.container,
+                    Entrypoint: '/opt/screwdriver/tini',
+                    Labels: {
+                        sdbuild: config.buildId
+                    },
+                    Cmd: [
+                        '--',
+                        '/opt/screwdriver/launch',
+                        '--api-uri',
+                        config.apiUri,
+                        '--emitter',
+                        '/opt/screwdriver/emitter',
+                        config.buildId
+                    ],
+                    Env: [
+                        `SD_TOKEN=${config.token}`
+                    ],
+                    HostConfig: {
+                        Memory: 2 * 1024 * 1024 * 1024,
+                        MemoryLimit: 3 * 1024 * 1024 * 1024,
+                        VolumesFrom: [
+                            `${launchContainer.id}:rw`
+                        ]
+                    }
+                })
+            ]))
+            .then(([logContainer, buildContainer]) => Promise.all([
+                this._startContainer(logContainer),
+                this._startContainer(buildContainer)
+            ]));
+    }
+
+    /**
+     * Stop a s3m build
+     * @method _stop
+     * @param  {Object}   config            A configuration object
+     * @param  {String}   config.buildId    ID for the build
+     * @return {Promise}
+     */
+    _stop(config) {
+        return this._findContainers(config.buildId)
+            .then(containers => Promise.all(containers.map(container =>
+                this._removeContainer(container))));
```

Comment:
Since we're passing force to it (it's in the function), it will stop and then remove it.